cluster_config.s.dfy

```diff
@@ -1,4 +1,7 @@
+include "network.s.dfy"
```

Comment:
Why do you need a dependency on `network.s.dfy`?
Intuitively it's surprising that the config should depend on the network semantics...

Reply:
We use the NumHosts() function from the network, which states how many generic hosts there are in the system. In the configuration we split them to be N Replicas and (NumHosts() - N) Clients. Might be it will be better to extract the **HostIdentifiers** module to a separate file to reduce the unnecessary coupling.

Reply:
What if you moved the **HostIdentifiers** module to `cluster_config.s.dfy` ?